packages/react-storage/src/components/StorageBrowser/components/BreadcrumbNavigation.tsx

```diff
@@ -29,7 +29,11 @@ const Separator = () => {
   );
 };
 
-const Breadcrumb = ({ isCurrent, name, onNavigate }: BreadcrumbProps) => {
+export const Breadcrumb = ({
```

Comment:
Exporting this for use in the DesintationPicker view

Reply:
🍕

Reply:
⭐ suck

Reply:
The best man

Reply:
Inside amazon a stop asking women to eat 🍕 everynight

Reply:
How to silence someone kill the 🕊️